src/types/models/index.ts

```diff
@@ -157,3 +157,11 @@ export interface IEmbed {
 		value: string;
 	}[];
 }
+
+export interface IStock extends IModel {
+	server_id: string;
+	user: Ref<IUser>;
+	symbol: string;
+	price: number;
+	currency: string;
```

Comment:
could probably make a `Currency` enum

Reply:
Hmmm, you mean to convert "`USD`" to "`$`", for example?

Reply:
Hmmm not sure I understand how an enum would be useful for this? Currency is one of the fields returned by the stock API - it's usually just "USD" as far as I've seen, but there are probably other possibilities dependent on the ticker symbol you look up.